scripts/autoshutdown.cron.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# This script is intended to be called once per minute. If it doesn't detect
```

Comment:
Doesn't really matter, but going full circle, back to the original `check && sleep 10m && check && shutdown -h now` would remove the dependence/strong coupling to the calling frequency.